src/user/entities/user.entity.ts

```diff
@@ -24,4 +32,10 @@ export class User {
   @Unique('email', ['email'])
   @Column({ length: 200 })
   email: string;
+
+  @CreateDateColumn({ name: 'createdAt', nullable: true })
+  createdAt: Timestamp;
```

Comment:
```suggestion
  createdAt: Timestamp;
```
We can keep this as `string` as well if need be. :thinking: